### DISCUSSION TITLE
Вставка из буфера обмена

### DISCUSSION THREAD

**Discussion by Spyker086** [totumonline/totum-ru-issues-and-discussions#464] [Есть идея, а что если...]
Добрый день! Подскажите как в текстовое поле вставить значение из буфера обмена без "ctrl + V". При редактировании появляется меня с кнопками "сохранить", "отменить". Можно ли в него добавить кнопку "вставить" ?

**Comment by ttm-alexeypolunin** [comment]
Врядли мы будем такое добавлять, очень редкая задача и пользователи скорее всего даже не поймут, че там за кнопка непонятная.

**Reply by Spyker086** [reply]
У нас есть пользователи, которые просят такую возможность. Ну что ж, ок
